runtime/src/accounts_db.rs

```diff
@@ -6019,6 +6033,11 @@ impl AccountsDb {
                     stored_account,
                 },
             )| {
+                if solana_address_map_program::check_id(&stored_account.account_meta.owner) {
+                    self.address_map_cache
+                        .populate_cache(&stored_account.meta.pubkey, stored_account.data);
+                }
```

Comment:
@jeffwashington as discussed, this is where I stuck in the address map cache changes. My understanding is that this will pick up every stored address map account from the snapshot.

Reply:
Seems reasonable to populate `address_map_cache` like this.